libparsec/crates/client/tests/unit/invite/bootstrap_organization.rs

```diff
@@ -176,7 +175,7 @@ async fn bad_token(env: &TestbedEnv) {
     let bootstrap_addr = ParsecOrganizationBootstrapAddr::new(
         env.server_addr.clone(),
         env.organization_id.clone(),
-        Some("<invalid token>".to_owned()),
+        None,
```

Comment:
For me, the goal of this test is to validate that we get `InvalidToken` when the provided token is invalid, here it's just missing.

Just have as to be different than `672bc6ba9c43455da28344e975dc72b7`